seed/static/seed/js/controllers/inventory_list_controller.js

```diff
@@ -92,10 +94,73 @@ angular.module('BE.seed.controller.inventory_list', [])
       } else {
         // No profiles exist
         $scope.columns = _.reject(all_columns, 'is_extra_data');
-      }
+      };
 
       $scope.restoring = false;
 
+      // Find labels that should be displayed and organize by applied id
+      $scope.show_labels = [];
```

Comment:
took me a sec to figure this out, how would you feel about changing this variable name to `$scope.labels_by_inventory_id` or something like that? A bit tough to infer what this is stored. Or maybe changing the comment above from `organize by applied id` to `organize by inventory id`?

Reply:
Renamed to "show_labels_by_inventory_id" for clarification.